engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,21 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+	#ifdef USE_TTS
+		Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+
+		Common::String result;
+
+		result+=story[0];
```

Comment:
Formatting doesn't match coding guidelines

Reply:
Thanks! I have a hard time understanding what is expected in this case though. The code conventions I have consulted are here: https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions

Reply:
`result += story[0];`